w3lib/url.py

```diff
@@ -7,8 +7,11 @@
 import posixpath
 import warnings
 import six
-from six import moves
-from w3lib.util import unicode_to_str
+from six.moves.urllib.parse import (urljoin, urlsplit, urlunsplit,
+                                    urldefrag, urlencode, urlparse,
+                                    quote, parse_qs, parse_qsl)
+from six.moves.urllib.request import pathname2url, url2pathname
+from w3lib.util import to_bytes, to_native_str, to_unicode
```

Comment:
I wonder if it is a time to add `__all__` to w3lib.url - see https://github.com/redapple/w3lib/commit/7d83b092f4054ba439a115ec041e9b190385b155 and https://github.com/scrapy/scrapy/blob/ebef6d7c6dd8922210db8a4a44f48fe27ee0cd16/scrapy/utils/url.py#L16;